crates/l1tx/src/messages.rs

```diff
@@ -18,20 +18,20 @@ pub enum L1Event {
     GenesisVerificationState(u64, HeaderVerificationState),
 }
 
-/// Core protocol specific transaction. It can be thought of as relevant transactions for the
-/// Protocol
+/// Core protocol specific bitcoin transaction reference. A bitcoin transaction can have multiple
+/// operations relevant to protocol. This is used in the context of [`BlockData`].
 #[derive(Clone, Debug, BorshSerialize, BorshDeserialize)]
 pub struct ProtocolOpTxRef {
```

Comment:
The name of this type doesn't really accurately describe what it represents at this point.  Perhaps a name like `ProtocolTxEntry` now.

Reply:
Good point, the name is not accurately reflective of what it holds. I would go for `ProtocolOpsTxRef` because it still is a transaction ref and contains multiple `ProtocolOp`s now.

Reply:
I would go for something like `ProtocolTxEntry` since it doesn't refer to a tx on it own like `TxRef` does (as that includes the block height).  This type doesn't make sense to include the block height since it's already being presented as the contents of a block and all of the other entries in the list would have the same block height.